apps/server/src/modules/tool/controller/tool.controller.ts

```diff
@@ -27,4 +31,12 @@ export class ToolController {
 		const mapped: Lti11LaunchResponse = this.lti11ResponseMapper.mapAuthorizationToResponse(authorization);
 		return mapped;
 	}
+
+	@Post('externalTool')
+	async createExternalTool(
+		@Body() externalToolParams: ExternalToolParams,
```

Comment:
Maybe better "ExternalToolBody" or something like this. Because you are using it as payload. Params could be a Query Parameter or a Path Parameter